### DISCUSSION TITLE
Welcome to Termgine Discussions!

### DISCUSSION THREAD

**Discussion by Morasiu** [Morasiu/Termgine#7] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by phoenix-iv** [comment]
Hi, I'm a long time lurker and user of GitHub projects.   I would like to contribute to this project by writing some code, but I'm not sure where to start.  I went on the Trello board to look for tasks, but I didn't find any way to assign them to myself.  Is there anything I can help with?
